Tasks/GradleV2/task.json

```diff
@@ -363,6 +363,9 @@
         "LocateJVMBasedOnVersionAndArch": "Locate JAVA_HOME for Java %s %s",
         "UnsupportedJdkWarning": "JDK 9 and JDK 10 are out of support. Please switch to a later version in your project and pipeline. Attempting to build with JDK 11...",
         "FailedToLocateSpecifiedJVM": "Failed to find the specified JDK version. Please ensure the specified JDK version is installed on the agent and the environment variable '%s' exists and is set to the location of a corresponding JDK or use the [Java Tool Installer](https://go.microsoft.com/fwlink/?linkid=875287) task to install the desired JDK.",
-        "NoCodeCoverage": "No code coverage results were found to publish."
+        "NoCodeCoverage": "No code coverage results were found to publish.",
+        "InvalidBuildFile": "Invalid or unsupported build file",
```

Comment:
I don't see `InvalidBuildFile` used at all by the gradle CC.

Reply:
These are part of the common CC module. Right now we need to duplicate all the resources that might be used when the helpers are called.